.travis.yml

```
@@ -82,7 +82,7 @@ script: |
         MJ_TIMEOUT=60 MJ_RUN=target/release/compiler-cli python3 ./mjtest/mjt.py lexer && \
         MJ_TIMEOUT=60 MJ_RUN=target/release/compiler-cli python3 ./mjtest/mjt.py syntax && \
         MJ_TIMEOUT=60 MJ_RUN=target/release/compiler-cli python3 ./mjtest/mjt.py semantic && \
-        MJ_TIMEOUT=60 MJ_RUN=target/release/compiler-cli python3 ./mjtest/mjt.py compile-firm ;
+        MJ_TIMEOUT=600 MJ_RUN=target/release/compiler-cli python3 ./mjtest/mjt.py compile-firm ;
```

This needs to be <10 min for travis to successfully fail. 
```suggestion
        MJ_TIMEOUT=500 MJ_RUN=target/release/compiler-cli python3 ./mjtest/mjt.py compile-firm ;
```